packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/header-mixed.twig

```diff
@@ -50,7 +50,11 @@
 
       <div class="ma__header__hamburger__nav-container" aria-hidden="true">
         <div class="ma__header__hamburger__logo ma__header__hamburger__logo--mobile">
-          {% include "@atoms/09-media/site-logo.twig" %}
+          {% include "@atoms/09-media/site-logo.twig" with {

```

Comment:
Can you please revert the data changes? This changed every page in VRT 
Also site level data is set here packages/patternlab/styleguide/source/_data/data.json